tests/test_runner/test_parallel.py

```diff
@@ -44,6 +55,17 @@ def dummy_test(self):
 
 class RemoteTestResultTest(SimpleTestCase):
 
+    def test_pickle_errors_detection(self):
+        picklable_error = RuntimeError('This is fine')
+        not_unpicklable_error = ExceptionThatFailsUnpickling('one arg', 'another arg')
+
+        result = RemoteTestResult()
+        result._confirm_picklable(picklable_error)
+
+        # __init__() missing 1 required positional argument (different on PY2)
+        with self.assertRaisesMessage(TypeError, 'argument'):
```

Comment:
I would be in favor of including the full exception text for the two cases with `if PY2`, etc. It's more cryptic than it needs to be. And when switching to Django 2, the `PY2` block can simply be deleted rather than having to remember to fill in the fuller message.